simde/arm/neon/rshr_n.h

```diff
@@ -48,6 +48,13 @@ simde_x_vrshrs_n_s32(int32_t a, const int n)
   return (a >> ((n == 32) ? 31 : n)) + ((a & HEDLEY_STATIC_CAST(int32_t, UINT32_C(1) << (n - 1))) != 0);
 }
 
+SIMDE_FUNCTION_ATTRIBUTES
+uint32_t
+simde_x_vrshrs_n_u32(uint32_t a, const int n)
+    SIMDE_REQUIRE_CONSTANT_RANGE(n, 1, 32){
```

Comment:
You're missing a space between the `)` and `{` at the end of the line.  I'll fix it, just wanted you to see it.